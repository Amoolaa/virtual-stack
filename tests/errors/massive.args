"tests/errors/massive.x2017"
