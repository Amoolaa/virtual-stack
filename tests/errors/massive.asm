FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
FUNC LABEL 0
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5  
    MOV STK A VAL 5
    MOV STK A VAL 5
    MOV STK A VAL 5
    RET
