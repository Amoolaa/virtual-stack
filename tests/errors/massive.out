Binary file is too large.
